.github/actions/spelling/patterns.txt

```diff
@@ -1,142 +1,216 @@
 # See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns
 
-# Automatically suggested patterns
-# hit-count: 1262 file-count: 4
-# ANSI color codes
-(?:\\(?:u00|x)1[Bb]|\x1b|\\u\{1[Bb]\})\[\d+(?:;\d+|)m
+^(?:Build|)Requires: [-.\w]+
 
-# hit-count: 743 file-count: 13
-# hex runs
-\b[0-9a-fA-F]{16,}\b
+"\w+"(?= instead of ")
 
-# hit-count: 739 file-count: 6
-# base64 encoded content, possibly wrapped in mime
-(?:^|[\s=;:?])[-a-zA-Z=;:/0-9+]{50,}(?:[\s=;:?]|$)
+NO_VOWELS:\s+'.+'
 
-# hit-count: 215 file-count: 101
-# https/http/file urls
-(?:\b(?:https?|ftp|file)://)[-A-Za-z0-9+&@#/%?=~_|!:,.;]+[-A-Za-z0-9+&@#/%=~_|]
+\b(?:e2e|k3s)(?=[a-z])
+
+HTPASSWD='.+'
+
+\b(?:azure|docker|kontainer)(?=[a-z])
+
+\b(?:cluster|node)(?=[a-z]{2})
+
+# *.spec.ts
+doRequest\('.+?'
```

Comment:
Surprised we need this; the only hit I expect is https://github.com/rancher-sandbox/rancher-desktop/blob/f26afeaf92e2712bab90380c31e5728b4003053b/e2e/rdctl.e2e.spec.ts#L288 and that can be worked around by changing the nonsense string instead.

Reply:
Happy to change the nonsense string!